src/app/system/manage-surveys/manage-surveys.component.html

```diff
@@ -0,0 +1,79 @@
+<div class="container m-b-20" fxLayout="row" fxLayoutAlign="end" fxLayoutGap="20px">
+  <button mat-raised-button color="primary">
+    <fa-icon icon="plus"></fa-icon>&nbsp;&nbsp;
+    Create Surveys
+  </button>
+</div>
+  
+<div class="container">
+
+  <div fxLayout="row">
+    <mat-form-field fxFlex>
+      <mat-label>Filter</mat-label>
+      <input matInput (keyup)="applyFilter($event.target.value)">
+    </mat-form-field>
+  </div>
+  
+  <div class="mat-elevation-z8">
+    <table mat-table [dataSource]="dataSource" matSort>
+
+      <!-- Key Column -->
+      <ng-container matColumnDef="key">
+          <th mat-header-cell *matHeaderCellDef mat-sort-header='key'> Key </th>
+          <td mat-cell *matCellDef="let survey"> 
```

Comment:
Keep all the content related to `<td>` on a single line if its just a value of an object.